source/gui/addonStoreGui/controls/details.py

```diff
@@ -264,6 +266,19 @@ def _refresh(self):
 						pgettext("addonStore", "Installed version:"),
 						details._addonHandlerModel.version
 					)
+
+					self._appendDetailsLabelValue(
+						# Translators: Label for an extra detail field for the selected add-on. In the add-on store dialog.
+						pgettext("addonStore", "Minimum NVDA version:"),
+						formatVersionForGUI(*details._addonHandlerModel.minimumNVDAVersion)

```

Comment:
This field should exist on details

```suggestion
						formatVersionForGUI(*details.minimumNVDAVersion)
```